### PR TITLE
feat!: move tools to autoware_tools repository

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

Moved the following packages to https://github.com/autowarefoundation/autoware_tools
- accel_brake_map_calibrator
- planning_debug_tools
- simulator_compatibility_test
- tier4_debug_tools

https://github.com/autowarefoundation/autoware_tools/pull/8

## Tests performed

build on CI

## Effects on system behavior

Nothing

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by shmpwk — APPROVED** [review]
LGTM

**Review by TakaHoribe — APPROVED** [review]

**Comment by takayuki5168** [comment]
cc @Soohyeok-Park @SoohyeokPark-MORAI

File: planning/.pages [hunk 1]
```diff
@@ -77,8 +77,8 @@ nav:
   - 'Additional Tools':
     - 'RTC Replayer': planning/rtc_replayer
     - 'Planning Debug Tools':
-      - 'About Planning Debug Tools': planning/planning_debug_tools
-      - 'Stop Reason Visualizer': planning/planning_debug_tools/doc-stop-reason-visualizer
+      - ['About Planning Debug Tools'](https://github.com/autowarefoundation/autoware_tools/tree/main/planning_debug_tools)
+      - ['Stop Reason Visualizer'](https://github.com/autowarefoundation/autoware_tools/tree/main/planning_debug_tools/doc-stop-reason-visualizer)
```
Comment (xmfcx) [inline planning/.pages:81]: This probably causes deploy-docs to fail.

Comment (takayuki5168) [reply planning/.pages:81]: Thank you for the report.
Seems it was already fixed.
https://github.com/autowarefoundation/autoware.universe/pull/6465/files

Comment (xmfcx) [reply planning/.pages:81]: Yes, created an issue but forgot to link it here:
- https://github.com/autowarefoundation/autoware.universe/issues/6508

**Pull request merged**